config.go

```diff
@@ -92,12 +92,19 @@ func parseAPKList(list string) []string {
 }
 
 func parseAppList(list string) (apps []string) {
-	for _, app := range strings.Split(list, "\n") {
-		apps = append(apps, strings.TrimSpace(app))
+	if list == "" {
```

Comment:
please have a trimspace on this string, also on other places you are checking string emptyness by len, not value